services/streamlit/src/app/streamlit_app.py

```diff
@@ -0,0 +1,150 @@
+import json
+import os
+
+import altair as alt
+import joblib
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import requests
+import streamlit as st
+from transformers import pipeline
+
+LABEL_MAPPING = {
+    "World": "World 🌎",
+    "Sports": "Sports 🏆",
+    "Business": "Business 💹",
+    "Sci/Tech": "Sci/Tech 📡",
+}
+
+DEFAULT_LAMBDA_URL = (
+    f"http://lambda:8080/2015-03-31/functions/function/invocations"
+)
+LAMBDA_URL = os.environ.get("LAMBDA_URL", DEFAULT_LAMBDA_URL)
+
+
+def request_to_lambda(
+    request_text: str, lambda_url: str = LAMBDA_URL
+) -> pd.DataFrame:
+    # curl -POST "http://localhost:9000/2015-03-31/functions/function/invocations" -d '{"text": "A test sentence."}'
+    data = json.dumps({"text": request_text})
+    response = requests.post(lambda_url, data=data).json()
+    body = response["body"]
+
+    df_response = (
+        pd.read_json(body)
+        .reset_index()
+        .rename(
+            {
+                "index": "label",
+                "predicted_labels": "score",
+            },
+            axis=1,
+        )
+    )
+    return df_response
+
+
+def pretty_print_label(label: str):
+    pretty_label = LABEL_MAPPING.get(label, "Undefined 😵‍💫")
+    return pretty_label
+
+
+def get_classifier(model_id):
+    return pipeline("text-classification", model=model_id)
+
+
+def classify_news(classifier, text):
+    preds = classifier(text, return_all_scores=False)
+    label = {0: "World", 1: "Sports", 2: "Business", 3: "Sci/Tech"}
+    text_label = label.get(int(preds[0]["label"][-1]))
+    preds = classifier(text, return_all_scores=True)
+    for items in preds[0]:
+        items["label"] = label.get(int(items["label"][-1]))
+
+    return text_label, preds[0]
```

Comment:
I think these are not needed anymore right?

Reply:
nope, removing!